### PR TITLE
REF: Use actionsheet anchor property button ipad options

### PR DESCRIPTION
**Pull request by marcosrdz**
Simulator Screen Shot - iPad Pro (12 9-inch) (4th generation) - 2021-02-26 at 18 57 30

instead of at the center, it anchors to the corresponding button

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/jaqayadbtzzptd4cfm3fg5w3b4

[download apk](http://filestorage.bluewallet.io:1488/anchor-1614386688.apk)

File: components/WalletsCarousel.js [hunk 1]
```diff
@@ -346,7 +346,7 @@ const WalletsCarousel = forwardRef((props, ref) => {
 
 WalletsCarousel.propTypes = {
   vertical: PropTypes.bool,
-  selectedWallet: PropTypes.object,
+  selectedWallet: PropTypes.string,
```
Comment (Overtorment) [inline components/WalletsCarousel.js:349]: hmm

**Review by Overtorment — APPROVED** [review]

File: components/WalletsCarousel.js (hunk 1 above)
Comment (marcosrdz) [reply components/WalletsCarousel.js:349]: Yeah. Removed a warning. Its supposed to be a string.

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/uvn1b6vcz8x98ecn309c2d4r4w

[download apk](http://filestorage.bluewallet.io:1488/anchor-1614452160.apk)

**Review by ncoelho — APPROVED** [review]

**Pull request merged**